server/channels/api4/post_test.go

```diff
@@ -3129,7 +3128,7 @@ func TestWebHubMembership(t *testing.T) {
 }
 
 func TestWebHubCloseConnOnDBFail(t *testing.T) {
-	t.Skip("https://mattermost.atlassian.net/browse/MM-61780")
+	t.Skip("MM-61780")
```

Comment:
1/5 that I like the link as that makes it easier to link and check who is working on the test.

Reply:
Haha okay, I see this isn't consistent. But I always have been using the short form rather than the full name. That's why went with this. I don't mind either ways, so going to leave it as is for now.